packages/angular/package.json

```diff
@@ -67,10 +67,10 @@
     "@nx/workspace": "file:../workspace"
   },
   "peerDependencies": {
-    "@angular-devkit/build-angular": ">= 15.0.0 < 18.0.0",
-    "@angular-devkit/schematics": ">= 15.0.0 < 18.0.0",
-    "@schematics/angular": ">= 15.0.0 < 18.0.0",
-    "@angular-devkit/core": ">= 15.0.0 < 18.0.0",
+    "@angular-devkit/build-angular": "~17.1.0-next.1",
+    "@angular-devkit/schematics": "~17.1.0-next.1",
+    "@schematics/angular": "~17.1.0-next.1",
+    "@angular-devkit/core": "~17.1.0-next.1",
```

Comment:
Does this impact our LTS support

Reply:
There's an issue with npm resolving prerelease versions in peer dependencies. We've had this issue in the past as well and I had it while doing this, but I'm gonna revert it and try again, because some of the changes made to our e2e might not surface it anymore.

It's an issue only with npm and peerDependencies with prerelease version ranges.

Reply:
Oh i've definitely encountered some of those issues. pre-release versions do have some issue, i remember this. That's fine :) just wanted to make sure it wasn't going to lead to further issues down the line

Reply:
I'll play a bit with some combinations and see. Otherwise, we'll make this change.